plugin-redfish/go.mod

```diff
@@ -3,11 +3,21 @@ module github.com/ODIM-Project/ODIM/plugin-redfish
 go 1.13
 
 require (
-	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20200727133207-df3dfb728bd1
-	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201015111324-83393fdedb82
+	github.com/Joker/hpp v1.0.0 // indirect
+	github.com/ODIM-Project/ODIM/lib-messagebus v0.0.0-20201201072448-9772421f1b55
+	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20201201072448-9772421f1b55
 	github.com/gofrs/uuid v3.2.0+incompatible
-	github.com/google/uuid v1.1.2-0.20200519141726-cb32006e483f
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
```

Comment:
why this package is added

Reply:
This package got added after the go mod tidy